### DISCUSSION TITLE
How to disable paste on click in the middle mouse button?

### DISCUSSION THREAD

**Discussion by Exadra37** [wezterm/wezterm#4114] [Q&A]
Cannot find in the docs a way of disabling the middle click paste. Is it possible?

**Comment by lwbt** [comment]
Based on this answer https://github.com/wez/wezterm/discussions/3760 the following should do it:

```lua
config.mouse_bindings = {
  {
    event = { Up = { streak = 1, button = "Middle" } },
    mods = "NONE",
    action = wezterm.action.Nop,
  },
}
```

**Reply by yttriumz** [reply]
At the time of version `20240203.110809.5046fc22`, according to [the doc](https://wezfurlong.org/wezterm/config/mouse.html#default-mouse-assignments), `Up` should be `Down`, i.e.:

```lua
config.mouse_bindings = {
  {
    event = { Down = { streak = 1, button = "Middle" } },
    mods = "NONE",
    action = wezterm.action.Nop,
  },
}
```
